packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Geojson/controlPanel.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t } from '@superset-ui/translation';
+import { validateNonEmpty, legacyValidateInteger } from '@superset-ui/validator';
+import { formatSelectOptions } from '../../utilities/utils';
+import { columnChoices } from '../../utilities/controls';
+import {
+  filterNulls,
+  jsColumns,
+  jsDataMutator,
+  jsTooltip,
+  jsOnclickHref,
+  fillColorPicker,
+  strokeColorPicker,
+  filled,
+  stroked,
+  extruded,
+  viewport,
+  mapboxStyle,
+} from '../../utilities/Shared_DeckGL';
+
+export default {
+  controlPanelSections: [
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'geojson',
+            config: {
+              type: 'SelectControl',
+              label: t('GeoJson Column'),
+              validators: [validateNonEmpty],
+              description: t('Select the geojson column'),
+              mapStateToProps: state => ({
+                choices: columnChoices(state.datasource),
+              }),
+            },
+          },
+          null,
+        ],
+        ['row_limit', filterNulls],
+        ['adhoc_filters'],
+      ],
+    },
+    {
+      label: t('Map'),
+      controlSetRows: [
+        [mapboxStyle, viewport],
+        // TODO [autozoom, null], // import { autozoom } from './Shared_DeckGL'
```

Comment:
Is this for future PR?

Reply:
I'm going to ask Evan if he has any context on this. But I believe so.

Reply:
@kristw The basic TODO was already there (not sure how old, but before my time). I added the `// import...` comment because I knew that control exists, and figured someone in the future might want to know about it.

If you know the context on the original feature (or lack thereof), I'm wondering if maybe we should just delete that TODO entirely.